### ISSUE TITLE
Upload team photo as part of submission process

### ISSUE THREAD

**Issue by rsgonzal** [Iridescent-CM/technovation-app#3222]
Clicking team photo from the submission page should take you directly to the summary section of "My Team" 

after saving your photo, it should bring you back to that same page

![Screen Shot 2022-01-10 at 5.13.41 PM.png](https://images.zenhubusercontent.com/60527169cde2772242d0de95/bd3d3a3f-e5a1-4ba8-889f-7aee3bd87a8a)

**Comment by dboyer** [comment]
We decided to integrate the team photo upload into the submission process rather than move the user between submissions and their dashboard to upload the photo. 

We will need to add a step in the submission process for uploading the photo. There is currently no section for "Start" which includes the honor code and team photo.

**Comment by dboyer** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Technovation-605271851956b1000ec2f24d/issues/Iridescent-CM/technovation-app/3222?planning-poker) with Zenhub @shaun-technovation @viviancan

**Comment by dboyer** [comment]
Edit: We have discovered this page exists. This ticket now covers having students go directly to the "start" section of the submission process.

**Comment by viviancan** [comment]
Confirming this is on QA

**Comment by dboyer** [comment]
On QA, I tested:

- Changed the team photo on an existing complete submission. The photo updated in the submission, on the project page, and on the team summary page on the dashboard
- Started a new submission. I was taken to the honor code and then to the overview page for the start section of the submission. I was able to select, edit, and upload an image using Filestack. The team photo part was checked off as complete.
- I went back to the profile and the uploaded photo was showing on the team summary photo page.
- Logged in as a new student and created a team. Uploaded a photo on the team summary page on my profile. Started a submission and my team photo was already there. I was able to update the team photo.
